collections/userEvents.coffee

```diff
@@ -61,3 +62,13 @@ Meteor.methods
         lastModifiedByUserId: user._id,
         lastModifiedByUserName: user.profile.name
       }})
+  updateUserEventArticleCount: (id, countModifier) ->
+    event = UserEvents.findOne(id)
```

Comment:
I think it would be better to require the admin role incase we want to add a type of account that doesn't have editing privileges.